pkg/reconciler/workload/resource/resource_controller.go

```diff
@@ -486,6 +486,17 @@ func (c *Controller) enqueueSyncTargetKey(syncTargetKey string) {
 	}
 }
 
+// isUpSynced returns true if the labels of the resource contain at least
+// one `ResourceState` label with the `Upsync` value.
+func isUpSynced(labels map[string]string) bool {
```

Comment:
as a special helper only used in one stop, please move it into the other file.

Reply:
fixed